demo/sections/components/ContextSidebar.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <ComponentPage title="Accordion" :demos="[{ title: 'Accordion' }]">
+    <template #accordion>
+      <PContextSidebar>
+        <template #header>
+          Header
+        </template>
+        <PContextNavItem to="#0" title="Context Nav Item" icon="Agent" />
+        <PContextNavItem to="/" title="Context Nav Item" />
+        <PContextNavItem to="/" title="Context Nav Item" />
+        <PContextNavItem to="/" title="Context Nav Item" />
+        <template #footer>
+          Footer
+        </template>
+      </PContextSidebar>
+    </template>
```

Comment:
I don't see a `#accordion` slot on the component page - perhaps this was copy-pasted in error?